build.xml

```diff
@@ -402,38 +402,42 @@
     </if>
 
     <!-- build and configure the requested database type -->
+    <phingcall target="setup_database" />
+
+    <!-- start Solr (use restart in case of old PID files) -->
+    <phingcall target="setup_solr" />
+
+    <phingcall target="import_biblios" />
+    <phingcall target="import_authorities" />
+    <phingcall target="build_alphabrowse" />
+  </target>
+
+  <!-- Setup basic configuration and run the install command -->
+  <target name="configure_install" description="configure and run installation">
+    <!-- fail if a local config.ini already exists -->
+    <property name="local_config_ini" value="${localdir}/config/vufind/config.ini" />
```

Comment:
We're doing the local_config_ini check in two places. Should we refactor to a separate task, or is there a reason to repeat it?

Reply:
Refactored. This makes the output a bit less clean as it seems as if the subsequent output belongs to the check task, so I named it so that the output doesn't look so odd.

In any case both are needed as I want to ensure that any extra_startup_command doesn't run if config exists, and it's done before configuration and install, which also needs to be prevented if config already exists.